lib/export_tables_to_big_query.rb

```diff
@@ -92,7 +92,7 @@ def bigquery_data(record, table)
       # being nullable.
       data = '' if c.name == 'hired_status' && data.nil?
       # This prevents an error whereby BigQuery rejects arrays of exactly one element.
-      data = data.to_s if c.name == 'job_roles'
+      data = data.to_s if c.is_a?(Array) && c.length == 1
       data = data.to_s if c.name == 'working_patterns'
```

Comment:
Can we lose this line as well?

Reply:
We can, indeed.

Reply:
Ah, wait, just realised why it wasn't done this way the first time. Once it's cast as a string, you can't then push an array to it and vice-versa.  Correcting.